mod_reforged/hooks/entity/tactical/player.nut

```diff
@@ -24,6 +24,12 @@
 			::Const.LevelXP.push(::Const.LevelXP.top() + 4000 + 1000 * (::Const.LevelXP.len() - 11));
 		}
 
+		// Temporary buff to vanilla drill sergeant until our Retinue Rework
+		if (!isScenarioMode && _scale && ::World.Retinue.hasFollower("follower.drill_sergeant"))
```

Comment:
`isScenarioMode` isn't defined in this function because it's a hook. So you have to look at the vanilla function for the conditions which define that boolean and use those conditions manually here.